anyway/widgets/suburban_widgets/suburban_crosswalk_widget.py

```diff
@@ -26,7 +26,9 @@ def generate_items(self) -> None:
         )
 
     @staticmethod
-    def get_crosswalk(road, start_time, end_time) -> None:
+    def get_crosswalk(
+        road, start_time, end_time
+    ) -> Dict[str, Union[List[Dict[str, Union[int, Any]]], Any]]:

```

Comment:
I think that `Union[List[Dict[str, Union[int, Any]]], Any]` is equivalent to `Any`

Reply:
makes sense, weird that pycharm didn't catch this